dvc/config_schema.py

```diff
@@ -179,10 +179,12 @@ class RelPath(str):
                 },
                 "hdfs": {"user": str, "kerb_ticket": str, **REMOTE_COMMON},
                 "webhdfs": {
-                    "hdfscli_config": str,
-                    "webhdfs_token": str,
-                    "user": str,
-                    "webhdfs_alias": str,
+                    "kerberos": Bool,
+                    "kerberos_principal": str,
+                    "proxy_to": str,
+                    "ssl_verify": Any(Bool, str),
+                    "token": str,
+                    "use_https": Bool,
```

Comment:
`use_https` - should it be swebhdfs per https://hadoop.apache.org/docs/r3.1.0/api/org/apache/hadoop/fs/SWebHdfs.html ? Or is the point that it enables HTTPS (SWebHdfs being an implementation detail)

Reply:
`ssl_verify` - if it's Hadoop specific should it be `hadoop_ssl_verify`?

Reply:
If my understanding of the Hadoop docs is correct, then `swebhdfs` protocol is `webhdfs` over `https`, but fsspec (and DVC) does not actually support `swebhdfs` protocol in URL strings and instead uses `webhdfs` with a `use_https` flag being true.

The question is if we want to create a new `swebhdfs` protocol to be consistent with the actual standard, or keep it as it is to avoid multiplying protocols on our end and just use this `use_https` flag.

As `fsspec` shows support for actually using `swebhdfs` in URL strings is rather inconsistent in the hadoop ecosystem it seems, so it wouldn't be crazy to just use `webhdfs` and then provide this flag.

Regarding the hadoop prefix see other comments.